op-mode-definitions/show-log.xml.in

```diff
@@ -398,12 +398,20 @@
             </properties>
             <command>journalctl --no-hostname --boot --unit snmpd.service</command>
           </leafNode>
-          <leafNode name="ssh">
+          <node name="ssh">
             <properties>
               <help>Show log for Secure Shell (SSH)</help>
             </properties>
             <command>journalctl --no-hostname --boot --unit ssh.service</command>
-          </leafNode>
+            <children>
+              <node name="dynamic-protection">
+                <properties>
+                  <help>Show dynamic-protection log</help>
+                </properties>
+                <command>journalctl --no-hostname -p info -t sshguard -o short</command>
```

Comment:
Any particular reason to diverge from the usual path by specifying the systemd unit to read from?

Should be: `journalctl --no-hostname --boot --unit sshguard.service` instead

Reply:
This was accepted like this on the 1.5 commit, does it need to change? The output is functionally the same. If so, like the sudo change, 1.5 would need to be updated as well.

Reply:
We have a lot of code accepted in 1.4 and 1.5 that we later iterate on. No big deal.

I will append a commit with the changes to this PR here